guardian/support-frontend
# Add feast benefit to checkout step 2 benefits list

## What are you doing in this PR?

Add a Feast benefit to the benefits list of step 2 of the checkout. #6136 updated the copy of the Feast offer on the landing page. This PR adds the benefit on the next step of the checkout.

This change is not behind the `feast` AB test so can only be merged once that test goes live (it's not actually an AB test as there is only one variant)
 
 


## How to test

Go to https://support.thegulocal.com/uk/contribute/checkout?selected-amount=12&selected-contribution-type=monthly&product=SupporterPlus, click "View Details" and see Feast benefit.




## Screenshots

Before
<img width="597" alt="image" src="https://github.com/guardian/support-frontend/assets/114918544/4a743915-831e-43ce-ab3f-c687b1247440">

After
<img width="593" alt="image" src="https://github.com/guardian/support-frontend/assets/114918544/0df41dde-ceda-43f1-857f-ff78eb2c8315">
